lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -150,6 +162,38 @@ def add_address(post, payment, options)
         }.compact
       end
 
+      def add_3ds_data(post, options = {})
+        return unless options[:execute_threed]
+
+        three_d_secure = options[:three_ds_2]
+        browser_info_3ds = three_d_secure[:browser_info]
+        payment_options = post[:paymentOption] ||= {}
+        card = payment_options[:card] ||= {}
+        card[:threeD] = {
+          v2AdditionalParams: {
+            challengeWindowSize: (options[:browser_size]),
+          challengePreference: '03'
+          },
+        browserDetails: {
+          acceptHeader: '*\\/*',
```

Comment:
is this a correct regular expression to match with `text/html,application/xhtml+xml` ❓,  Why not use directly the text/html,application/xhtml+xml" or a regular expression like this `/text\/html.*application\/xhtml\+xml/` instead of  `'*\\/*'` ❓